pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -48,6 +48,12 @@
   // Pinot controller leader
   PINOT_CONTROLLER_LEADER("PinotControllerLeader", true),
 
+  // Pinot controller partition leader
+  PINOT_CONTROLLER_PARTITION_LEADER("PinotControllerPartitionLeader", true),
+
+  // Pinot controller resource enabled
+  PINOT_LEAD_CONTROLLER_ENABLED("PinotLeadControllerEnabled", true),
```

Comment:
`PINOT_LEAD_CONTROLLER_RESOURCE_ENABLED("PinotLeadControllerResourceEnabled", true)` for clarity?

Reply:
done.